core/settings.py

```
@@ -0,0 +1,689 @@
+    )
+    RECAPTCHA_PUBLIC_KEY = values.SecretValue(
+        environ=True, environ_prefix=None, environ_name="RECAPTCHA_PUBLIC_KEY"
+    )
+    CRISPY_ALLOWED_TEMPLATE_PACKS = (
+        "bootstrap",
+        "uni_form",
+        "bootstrap5",
+        "bootstrap4",
+    )
+    CRISPY_TEMPLATE_PACK = "bootstrap5"
+    #!SECTION END - Application definition
```

Ruff (0.12.2)

Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)